sdk/python/kfp/dsl/_ops_group.py

```diff
@@ -218,9 +218,10 @@ def __init__(self,
                  loop_args: Union[_for_loop.ItemList,
                                   _pipeline_param.PipelineParam],
                  parallelism: int = None):
```

Comment:
Since this is an `int` field but permits `None`, the type annotation should be `Optional[int]`

```suggestion
                 parallelism: Optional[int] = None):
```

Reply:
Thanks for the catch!